.travis/pre_before_install.sh

```diff
@@ -29,6 +29,18 @@ then
   return 0
 fi
 
+# check if a test was added
+NEEDS_TEST=$(git diff --name-only $RANGE | grep -E 'feature|bugfix')
+CONTAINS_TEST=$(git diff --name-only $RANGE | grep -E '^test_')
+
+if [[ $(git log --format=medium --no-merges "$RANGE" | grep "\[notest\]") ]]
+then
+  echo "[notest] is present - skipping the check for the test requirement"
+elif [ -n $NEEDS_TEST ] && ![ -n $CONTAINS_TEST ]; then
+  echo "Every feature and bugfix should come with a test."
+  exit 1
+fi
+
```

Comment:
I had a similar approach at pulp_file and it didn't work as I expected: 
- https://github.com/pulp/pulp_file/pull/426
- https://travis-ci.com/github/pulp/pulp_file/jobs/382763671#L283

Could you add a changelog entry `.feature` to verify if it is working?

Reply:
I actually took the code from that PR, I will test with .feature entry

Reply:
the travis still does not work as expected, i need to look more into this.